frontend/src/components/SearchBar.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {Container, Typography} from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+
+export default function SearchBar({getInput}: {
+    getInput: Function
```

Comment:
Ich würde empfehlen das hier zu machen `onChange: (searchTerm: string) => void`

Dann kannst du in der OrdersPage, wo du die SearchBar benutzt, wahrscheinlich einfach das hier machen: 
`<SearchBar onChange={setInputForFilter}/>`

Und dann in der SearchBar unten beim input musst du dann einfach nur die OnChange methode mit dem value vom input aufrufen: `onChange={(event: React.ChangeEvent<HTMLInputElement>) => onChange(event.target.value)}`